testsuite/gnat2goto/tests/loop_array_range/loop_array_range.adb

```diff
@@ -8,5 +8,5 @@ begin
       Sum := Sum + My_Arr(I);
    end loop;
 
-   pragma Assert(Sum=6);
+   pragma Assert(Sum=10);
```

Comment:
Why change this?  Why not have both?  A test can still pass if CBMC says `VERIFICATION FAILED`.  It is better to have a mix of tests TBH.